components/common/Layout/Layout.tsx

```diff
@@ -83,10 +86,14 @@ const SidebarView: FC<{ sidebarView: string; closeSidebar(): any }> = ({
   )
 }
 
-const SidebarUI: FC = () => {
+const SidebarUI: FC<{ links: any }> = ({ links }) => {
```

Comment:
You can even extract the links to its own type or interface (or import it from other place) and use that type in here too